filemanager/azureblobmanager.go

```diff
@@ -39,60 +39,69 @@ func NewAzureBlobManager(config map[string]interface{}, log logger.Logger, defau
 	}, nil
 }
 
-func (manager *azureBlobManager) ListFilesWithPrefix(ctx context.Context, startAfter, prefix string, maxItems int64) ListSession {
+func (m *AzureBlobManager) ListFilesWithPrefix(ctx context.Context, startAfter, prefix string, maxItems int64) ListSession {
 	return &azureBlobListSession{
 		baseListSession: &baseListSession{
 			ctx:        ctx,
 			startAfter: startAfter,
 			prefix:     prefix,
 			maxItems:   maxItems,
 		},
-		manager: manager,
+		manager: m,
 	}
 }
 
 // Upload passed in file to Azure Blob Storage
-func (manager *azureBlobManager) Upload(ctx context.Context, file *os.File, prefixes ...string) (UploadedFile, error) {
-	containerURL, err := manager.getContainerURL()
+func (m *AzureBlobManager) Upload(ctx context.Context, file *os.File, prefixes ...string) (UploadedFile, error) {
+	fileName := path.Join(m.config.Prefix, path.Join(prefixes...), path.Base(file.Name()))
```

Comment:
[minor]
```suggestion
	filePath := path.Join(m.config.Prefix, path.Join(prefixes...), path.Base(file.Name()))
```
or objName for consistency